### PR TITLE
[New] Add Flux-Slider@1.4.4 w/ git auto-update

### LINKED ISSUE #11101: [Request] Add Flux-Slider
**Library name:** Flux-Slider
**Git repository url:** https://github.com/joelambert/Flux-Slider
**npm package name or url** (if there is one): 
**License (List them all if it's multiple):** MIT
**Official homepage:** http://www.joelambert.co.uk/flux/
**Wanna say something? Leave message here:** 

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/44293977-request-add-flux-slider?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by dakshshah96**
close #11101, cc @joelambert

Pull request for issue: #11101 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/joelambert/Flux-Slider
 * Official website (optional, not the repository): http://www.joelambert.co.uk/flux
 * NPM package url (optional):
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 74
   - Count of stars: 1,349
   - Count of forks: 291
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/Flux-Slider/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "url": "https://github.com/joelambert/Flux-Slider"
+  },
+  "license": "MIT",
+  "author": "Joe Lambert <@joelambert> (http://www.joelambert.co.uk/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/joelambert/Flux-Slider.git",
+    "fileMap": [
+      {
+        "basePath": "js",
```
Comment (extend1994) [inline ajax/libs/Flux-Slider/package.json]: It's not good to use `js` or `css` as a basePath. Please modify it as the following:
```
"basePath": "",
"files": [
  "js/flux*",
  "css/style.css"
]
```

**Comment by dakshshah96** [comment]
@extend1994 Done 👍

**Comment by dakshshah96** [comment]
@qwe19272375 Please add `in progress` label here.

**Comment by sufuf3** [comment]
@dakshshah96 Could you discuss with the author about the image files seem missing?
There are two files that `style.css` need use `img/flux_wp_new.png` and `img/flux_wp_logo.png`.
ref: https://github.com/joelambert/Flux-Slider/blob/v1.4.4/css/style.css
```background:url("img/flux_wp_new.png") top left no-repeat, url("img/flux_wp_logo.png") ```

But these two files are not under the `img` folder.
And we need include these two files.

Thank you.

**Comment by dakshshah96** [comment]
@PeterDaveHello @sufuf3 Awaiting author response.

ref: https://github.com/joelambert/Flux-Slider/issues/60

**Comment by sufuf3** [comment]
@dakshshah96 Thank you.
Next time: you could add `cdnjs/cdnjs#11223` into the discuss comment.
Then in the CDNJS PR will show the referenced issue link. 
This will be helpful to us to trace.
Thank you.

**Comment by dakshshah96** [comment]
@sufuf3 Can you please add `wait for response` label here?

**Comment by dakshshah96** [comment]
@sufuf3 Can you please add `wait for response` label here?

File: ajax/libs/Flux-Slider/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "author": "Joe Lambert <@joelambert> (http://www.joelambert.co.uk/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/joelambert/Flux-Slider.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/flux*",
+          "css/style.css"
```
Comment (sufuf3) [inline ajax/libs/Flux-Slider/package.json]: Let's remove `"css/style.css"`
Because this file is used by the library website, only.
Thank you.

**Comment by sufuf3** [comment]
@dakshshah96 
thanks for contribution.
Would you mind working on this and helping improve this PR?
If you have any question, please let us know.

Many thanks.

**Comment by sashberd** [comment]
@sufuf3 @PeterDaveHello @extend1994 I cannot rebase this PR to latest master. Can you help me with it or just close this PR

**Comment by extend1994** [comment]
@sashberd Here you go

**Comment by sashberd** [comment]
@extend1994 Thanks, but I have 45 PR to merge/review in my sparse-checkout, why not to review/merge them and only after that continue to problematic PR`s?

**Comment by sashberd** [comment]
@sufuf3 @extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@sufuf3 Are you talikng about right lib. I do not see zepto in commits files and also in folder and files struct

**Comment by extend1994** [comment]
@sashberd Check the above link

**Comment by extend1994** [comment]
@sashberd I didn't ask you to change anything, that's why I used "comment" instead of "request".
Just wanted to discuss should we **also** need the files I mentioned above, they are in the git repo, not in config.

**Comment by sashberd** [comment]
@extend1994 OK. Now I understand what do you mean. I do not know if those files are needed. @PeterDaveHello Please help

**Comment by PeterDaveHello** [comment]
@sashberd yes? what help do you need?

**Comment by sashberd** [comment]
With answer to @extend1994  [discussion](https://github.com/cdnjs/cdnjs/pull/11223#pullrequestreview-140151278)

**Comment by PeterDaveHello** [comment]
@sashberd tried to confirm with upstream yet?

**Pull request closed without merging**